main.go

```diff
@@ -363,6 +363,18 @@ func main() {
 		setupLog.Error(err, "unable to create controller", "controller", "Application")
 		os.Exit(1)
 	}
+
+	if err = (&capabilitiescontroller.RedisConfigCMReconciler{
```

Comment:
There is no need for a new controller. 

I propose a new approach: Use the existing apimanager_controller.  You will need to add new watcher for owned (with ownerrefs) configmaps. Maybe filtering only for redis configmaps using labels. Then change the mutator, which currently is "CreateOnlyMutator", to something more restrictive. 

If you implement a new controller, the apimanager controller creates the config map and another controller ensures nobody change it. Maybe better to keep all the logic in one controller.

Reply:
Done. Thank you